dask_mongo/tests/test_core.py

```diff
@@ -187,3 +189,40 @@ def test_read_mongo_chunksize(connection_kwargs):
         3,
         1,
     )
+
+
+def test_connection_pooling(connection_kwargs):
+    records = gen_data(size=10)
+    database = "test-db"
+    collection = "test-collection"
+
+    mongo_client = _get_client(connection_kwargs)
+    database_ = mongo_client.get_database(database)
+    database_[collection].insert_many(deepcopy(records))
+
+    for _ in range(3):
+        read_mongo(
+            database,
+            collection,
+            chunksize=5,
+            connection_kwargs=connection_kwargs,
+        )
+    assert _get_num_clients() == 1
+    connection_kwargs.update(
+        {"event_listeners": [CommandLogger()]})
+    read_mongo(
+        database,
+        collection,
+        chunksize=5,
+        connection_kwargs=connection_kwargs,
+    )
+    assert _get_num_clients() == 2
+    for i in range(1, 20):
```

Comment:
Instead of using `20`, can we write the length of this `for`-loop in terms of `_CACHE_SIZE`? That way if, in the future, the size of `_CACHE_SIZE` is increased to, for example `50`, this test will still test will still pass 

```suggestion
    for i in range(_CACHE_SIZE):
```